sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_patch.py

```diff
@@ -0,0 +1,903 @@
+                continuation_token,
+                cls=lambda pipeline_response, _, response_headers: pipeline_response,
+            )
+
+        callback = kwargs.pop("cls", deserialization_callback)
+        return cast(
+            DocumentTranslationLROPoller[ItemPaged[DocumentStatus]],
+            self._client.begin_start_translation(
+                body=_StartTranslationDetails(inputs=inputs),
+                polling=DocumentTranslationLROPollingMethod(
+                    timeout=polling_interval,
+                    lro_algorithms=[TranslationPolling()],
+                    cont_token_response=pipeline_response,
+                    **kwargs
+                ),
+                cls=callback,
+                continuation_token=continuation_token,
+                **kwargs
+            ),
+        )
+
+    @distributed_trace
+    def get_translation_status(self, translation_id: str, **kwargs: Any) -> TranslationStatus:
+        """Gets the status of the translation operation.
+
+        Includes the overall status, as well as a summary of
+        the documents that are being translated as part of that translation operation.
+
+        :param str translation_id: The translation operation ID.
+        :return: A TranslationStatus with information on the status of the translation operation.
+        :rtype: ~azure.ai.translation.document.TranslationStatus
+        :raises ~azure.core.exceptions.HttpResponseError or ~azure.core.exceptions.ResourceNotFoundError:
+        """
+
+        translation_status = self._client.get_translation_status(translation_id, **kwargs)
+        return TranslationStatus._from_generated(  # pylint: disable=protected-access
+            _TranslationStatus(translation_status)
+        )
+
+    @distributed_trace
+    def cancel_translation(self, translation_id: str, **kwargs: Any) -> None:
+        """Cancel a currently processing or queued translation operation.
+
+        A translation will not be canceled if it is already completed, failed, or canceling.
+        All documents that have completed translation will not be canceled and will be charged.
+        If possible, all pending documents will be canceled.
+
+        :param str translation_id: The translation operation ID.
+        :return: None
+        :rtype: None
+        :raises ~azure.core.exceptions.HttpResponseError or ~azure.core.exceptions.ResourceNotFoundError:
+        """
+
+        self._client.cancel_translation(translation_id, **kwargs)
+
+    @distributed_trace
+    def list_translation_statuses(
+        self,
+        *,
+        top: Optional[int] = None,
+        skip: Optional[int] = None,
+        translation_ids: Optional[List[str]] = None,
+        statuses: Optional[List[str]] = None,
+        created_after: Optional[Union[str, datetime.datetime]] = None,
+        created_before: Optional[Union[str, datetime.datetime]] = None,
+        orderby: Optional[List[str]] = None,
```

Comment:
Please revert all instances where `order_by` was changed to `orderby`. This is a breaking change.

Reply:
Done